specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -900,6 +900,191 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/inventoryPolicies/{blobInventoryPolicyName}": {
+      "get": {
+        "tags": [
+          "BlobInventoryPolicies"
+        ],
+        "operationId": "BlobInventoryPolicies_Get",
+        "description": "Gets the blob inventory policy associated with the specified storage account.",
+        "x-ms-examples": {
+          "StorageAccountGetBlobInventoryPolicy": {
+            "$ref": "./examples/StorageAccountGetBlobInventoryPolicy.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
```

Comment:
why not use the common parameter for resource group name here (as in the parameters below)?

Reply:
WIll send another PR, as this is the same pattern across all SRP APIs